docs/quickstart-make.md

```diff
@@ -84,7 +84,7 @@ cluster creation complete!
 ![civostack cluster create complete](../docs/images/6-civostack-cluster-create-complete.png)
 
 ```sh
-bash virtual-datacenter/destroy.sh
+bash colony-vagrant/virtual-datacenter/destroy.sh
```

Comment:
Same as above re: location of the folder.

Reply:
colony-vagrant is related to the project root; virtual-datacenter is not in the root!